apps/explorer/src/assets/SVGIcons/CheckFill.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="#2dd7a7" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I've been trying to make icons configurable via things like CSS variables, so we could potentially change the fill so that it's use-settable, but not something we need to do now.